src/tools/info/ucx_info.c

```diff
@@ -27,6 +28,7 @@ static void usage() {
     printf("                'r' : remote memory access\n");
     printf("                't' : tag matching \n");
     printf("                'w' : wakeup\n");
+    printf("  -n         Estimated UCP endpoint count (for ucp_init).\n");
```

Comment:
minor: there is no dot in the end of other one sentence lines of this help message. So does not look consistent.